docs/essentials/controls.md

```diff
@@ -188,6 +188,29 @@ And here's what the resulting UI looks like:
 
 ![Controls addon expanded](./addon-controls-expanded.png)
 
+## Hide controls for certain fields on a particular story
+
+The `argTypes` annotation can be used to hide controls for a particular row, or even hide rows.
+
+Suppose you have a `Button` component with `borderWidth` and `label` properties (auto-generated or otherwise) and you want to hide the `borderWidth` row completely and disable controls for the `label` row on a specific story. Here's how you'd do that:
+
+```js
+import { Button } from 'button';
+
+export default {
+  title: 'Button',
+  component: Button,
+};
+
+export const CustomControls = (args) => <Button {...args} />;
+CustomControls.argTypes = {
+  borderWidth: { table: { disable: true } },
+  label: { control: { disable: true } },
+};
+```
```

Comment:
@novellac if you don't mind a couple of things regarding the snippet:
- This would be probably better suited for a snippet file, i'll leave the naming to you. And as this is based on example in the README. It's for React, so if you don't mind place it inside that folder.
- We can make a adjustment to the snippet and go with:
```js
// Button.stories.js

import React from 'react';
import { Button } from './Button';

export default {
  title: 'Button',
  component: Button,
};

const Template = (args) => <Button {...args}/>;

export const CustomControls= Template.bind({});
CustomControls.argTypes = {
 // hides the property in the UI.
 borderWidth: { table: { disable: true } },
  // disables controls for the label in the UI
  label: { control: { disable: true } },
};
```

Reply:
Sure I can make this a snippet. I'll also try to figure out how to make a Vue snippet for this (though I might need some help).

Reply:
Let me know once the snippets are up and we go from there.

Reply:
@novellac just a friendly reminder if you're willing to address the feedback that i've mentioned. We could do without the vue snippet for now and work on it later with a follow up pull request. 

Let me know when you've addressed this.

Stay safe